js/localization/messages/tr.json

```
@@ -0,0 +1,261 @@
+        "dxPivotGrid-rowFields": "Satır Alanları",
+        "dxPivotGrid-columnFields": "Sütun Alanları",
+        "dxPivotGrid-dataFields": "Veri Alanları",
+        "dxPivotGrid-filterFields": "Filtre Alanları",
+        "dxPivotGrid-allFields": "Tüm Alanlar",
+        "dxPivotGrid-columnFieldArea": "Sütun Alanlarını Buraya Bırak",
+        "dxPivotGrid-dataFieldArea": "Veri Alanlarını Buraya Bırak",
+        "dxPivotGrid-rowFieldArea": "Satır Alanlarını Buraya Bırak",
+        "dxPivotGrid-filterFieldArea": "Filtre Alanlarını Buraya Bırak",
+        "dxScheduler-editorLabelTitle": "Konu",
+        "dxScheduler-editorLabelStartDate": "Başlangıç Tarihi",
+        "dxScheduler-editorLabelEndDate": "Bitiş Tarihi",
+        "dxScheduler-editorLabelDescription": "Açıklama",
+        "dxScheduler-editorLabelRecurrence": "Tekrar",
+        "dxScheduler-openAppointment": "Randevu Aç",
+        "dxScheduler-recurrenceNever": "Asla",
+        "dxScheduler-recurrenceDaily": "Günlük",
+        "dxScheduler-recurrenceWeekly": "Haftalık",
+        "dxScheduler-recurrenceMonthly": "Aylık",
+        "dxScheduler-recurrenceYearly": "Yıllık",
+        "dxScheduler-recurrenceRepeatEvery": "Her tekrarla",
+        "dxScheduler-recurrenceRepeatOn": "Tekrarla",
+        "dxScheduler-recurrenceEnd": "Tekrarı bitir",
+        "dxScheduler-recurrenceAfter": "Sonra",
+        "dxScheduler-recurrenceOn": "İle",
+        "dxScheduler-recurrenceRepeatDaily": "günler",
+        "dxScheduler-recurrenceRepeatWeekly": "haftalar",
+        "dxScheduler-recurrenceRepeatMonthly": "aylar",
+        "dxScheduler-recurrenceRepeatYearly": "yıllar",
+        "dxScheduler-switcherDay": "Gün",
+        "dxScheduler-switcherWeek": "Hafta",
+        "dxScheduler-switcherWorkWeek": "Çalışma Haftası",
+        "dxScheduler-switcherMonth": "Ay",
+        "dxScheduler-switcherAgenda": "Ajanda",
+        "dxScheduler-switcherTimelineDay": "Zaman Çizelgesi Günü",
+        "dxScheduler-switcherTimelineWeek": "Zaman Çizelgesi Haftası",
+        "dxScheduler-switcherTimelineWorkWeek": "Zaman Çizelgesi Çalışma Haftası",
+        "dxScheduler-switcherTimelineMonth": "TZaman Çizelgesi Çalışma Ayı",
+        "dxScheduler-recurrenceRepeatOnDate": "tarihinde",
+        "dxScheduler-recurrenceRepeatCount": "olaylar",
+        "dxScheduler-allDay": "Tüm gün",
+        "dxScheduler-confirmRecurrenceEditMessage": "Yalnızca bu randevuyu veya tüm diziyi düzenlemek ister misiniz?",
+        "dxScheduler-confirmRecurrenceDeleteMessage": "Yalnızca bu randevuyu veya tüm diziyi silmek istiyor musunuz?",
+        "dxScheduler-confirmRecurrenceEditSeries": "Serileri düzenle",
+        "dxScheduler-confirmRecurrenceDeleteSeries": "Serileri sil",
+        "dxScheduler-confirmRecurrenceEditOccurrence": "Randevuyu düzenle",
+        "dxScheduler-confirmRecurrenceDeleteOccurrence": "Randevuyu sil",
+        "dxScheduler-noTimezoneTitle": "Saat dilimi yok",
+        "dxScheduler-moreAppointments": "{0} data",
```

I think there is a typo:
```suggestion
        "dxScheduler-moreAppointments": "{0} daha",
```